cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -0,0 +1,19 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+===============================
+Transaction Service Maintenance
+===============================
+
+.. highlight:: console
+
+The Transaction Service keeps track of all invalid transactions to exclude their writes from all future reads. 
+Over time this invalid list can grow and lead to performance degradation. To avoid that, you can prune the invalid 
+list after the data of invalid transactions has been removed---which happens during major HBase compactions of the 
+transactional tables.
+
+After a major compaction that started at time ``n`` completes, the invalid transaction list can be safely pruned
+until ``n - 1 day``, using the REST call at [Insert ref to REST API]
```

Comment:
```
After a major compaction that started at time ``n`` completes, the invalid transaction list can be safely pruned
until ``n -1 day``, using the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactions>`.
```
Given that the API is milliseconds, what is `n -1 day`? Should it be `n -86400000`? Or does `day` get converted?